cranelift-codegen/src/ir/mod.rs

```diff
@@ -49,7 +49,8 @@ pub use crate::ir::instructions::{
 };
 pub use crate::ir::jumptable::JumpTableData;
 pub use crate::ir::layout::Layout;
-pub use crate::ir::libcall::{get_libcall_funcref, get_probestack_funcref, LibCall};
+pub(crate) use crate::ir::libcall::get_libcall_funcref;
```

Comment:
Makes sense to me. Neither of Spidermonkey or Wasmtime seems to be using it, so it seems fine to make pub(crate) only. In this case, can you even remove it from this list, and have the verifier directly import it please?

Reply:
Sure, I made the change. That requires making libcall `pub(crate)`, not sure if that's intended.